src/App.jsx

```diff
@@ -14,33 +15,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [activeSort, setActiveSort] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const SORT_BY_ALPHABET = 'Alphabetically';
+  const SORT_BY_LENGTH = 'ByLength';
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  const sortAlphabetically = () => {
+    const sortedGoods = [...goods].sort((a, b) => {
+      return isReversed ? b.localeCompare(a) : a.localeCompare(b);
+    });
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+    setGoods(sortedGoods);
+    setActiveSort(SORT_BY_ALPHABET);
+  };
+
+  const sortByLength = () => {
```

Comment:
It would be great to create a universal function for sorting which accepts the sortType and return the sorted goods